developer/src/kmc-copy/test/test-copier.ts

```diff
@@ -242,12 +242,64 @@ describe('KeymanProjectCopier', function() {
       'source/burushos.bsk.burushaski.model.kps': 'expected/burushos.bsk.burushaski/sil.bsk.my_version.model.kps',
     }
   },
+
+  //
+  // jawa
+  //
+  {
+    title: 'should copy a v1.0 keyboard from Keyman Cloud',
+    new: 'my_jawa',
+    old: 'cloud:jawa',
+    cloud_path: '', ///release/j/jawa',
+    old_project: 'jawa.kpj',
+    expected_artifacts: {
+      "/release/j/jawa/LICENSE.md": "LICENSE.md",
+      "/release/j/jawa/jawa.kpj": "my_jawa.kpj",
+      "/release/j/jawa/build/jawa.js": "build/my_jawa.js",
+      "/release/j/jawa/build/jawa.kmx": "build/my_jawa.kmx",
+      "/release/j/jawa/build/jawa.kvk": "build/my_jawa.kvk",
+      // note: .kmp is missing because not referenced, this is okay (will be built)
+      "/release/j/jawa/source/jawa-help.htm": "source/jawa-help.htm",
+      "/release/j/jawa/source/jawa.ico": "source/my_jawa.ico",
+      "/release/j/jawa/source/jawa.keyman-touch-layout": "source/my_jawa.keyman-touch-layout",
+      "/release/j/jawa/source/jawa.kmn": "source/my_jawa.kmn",
+      "/release/j/jawa/source/jawa.kps": "source/my_jawa.kps",
+      "/release/j/jawa/source/jawa.kvks": "source/my_jawa.kvks",
+      "/release/j/jawa/source/jawa2.bmp": "source/jawa2.bmp",
+      "/release/j/jawa/source/readme.htm": "source/readme.htm",
+      "/release/j/jawa/source/welcome/welcome.htm": "source/welcome/welcome.htm",
+      "/release/shared/fonts/noto/Java/NotoSansJavanese-Bold.ttf": "external/release/shared/fonts/noto/Java/NotoSansJavanese-Bold.ttf",
+      "/release/shared/fonts/noto/Java/NotoSansJavanese-Regular.ttf": "external/release/shared/fonts/noto/Java/NotoSansJavanese-Regular.ttf",
+      // NOTE: the following files are not included because this is a 1.0
+      // project and they are not referenced by any files:
+      // * HISTORY.md
+      // * README.md
+      // * build/jawa.kmp
+    },
+    fixtures: {
+      // TODO-COPY
+    }
+  },
+
+
 ];
 
   for(const test of tests) {
     it(test.title, async function() {
+      if(TEST_SAVE_FIXTURES) {
+        this.timeout(120000);
+      }
+
       const outPath = path.join(outputRoot, test.new).replaceAll(/\\/g, '/');
-      const source = makePathToFixture('projects', test.old).replaceAll(/\\/g, '/') + '/';
+      const source = test.old.includes(':')
+        ? test.old
+        : makePathToFixture('projects', test.old).replaceAll(/\\/g, '/') + '/';
```

Comment:
Does resolveFilename take care of the path separators?

Reply:
Yes, I believe so